client/asset/dcr/dcr.go

```diff
@@ -748,7 +756,13 @@ func (dcr *ExchangeWallet) feeRate(confTarget uint64) (uint64, error) {
 	}
 	estimatedFeeRate, err := feeEstimator.EstimateSmartFeeRate(dcr.ctx, int64(confTarget), chainjson.EstimateSmartFeeConservative)
 	if err != nil {
-		return 0, err
+		if externalApiAllowed {
+			estimatedFeeRate, err = externalFeeEstimator(confTarget)
+			if err != nil {
+				dcr.log.Errorf("Failed to get fee rate from external API: %v", err)
+				return 0, err
+			}
+		}
```

Comment:
the original error that got us here is ignored, looks like

Reply:
I am checking for error now.